File: spring-boot-web/oauth2/src/main/java/org/luvx/boot/oauth2/Application.java
package org.luvx.boot.oauth2;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.web.embedded.tomcat.TomcatServletWebServerFactory;
import org.springframework.context.annotation.Bean;

@SpringBootApplication
public class Application {
    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }

    /**
     * 新启动一个服务
     */
    // @Bean
    // public TomcatServletWebServerFactory servletContainer() {
    //     return new TomcatServletWebServerFactory(8081);
    // }
}

File: spring-boot-kotlin/src/main/kotlin/org/luvx/kotlin/Application.kt
package org.luvx.kotlin

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.ComponentScan

private val log = mu.KotlinLogging.logger {}

@ComponentScan(basePackages = ["org.luvx.kotlin", "org.luvx.java"])
@SpringBootApplication
class Application

fun main(args: Array<String>) {
    runApplication<Application>(*args)
    log.info { "启动成功......" }
}
